### ISSUE TITLE
missing textures 

### ISSUE THREAD

**Issue by kotelkonrad** [EmosewaPixel/GregicAdditions#61]
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/ga_compressed.grout.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/shape.sword_blade.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/ga_compressed.coke.clay.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/mica_insulator_foil.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/ga_compressed.fireclay.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/shape.shovel_head.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/shape.axe_head.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/shape.hoe_head.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
textures/items/metaitems/mold.form.small_gear.png
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
-------------------------
08:46:45
FML.TEXTURE_ERRORS
Client thread
error
No other errors exist for domain gregtech
[latest.log](https://github.com/EmosewaPixel/GregicAdditions/files/3093106/latest.log)

**Comment by BlueCreeper160** [comment]
Yes, these textures aren't implemented yet. Is possible add them?

**Comment by ghost** [comment]
This project is abandoned by pixel if you want GA in latest GTCE I suggest switching to Shadows Of Greg witch is a fork of GA
